lib/newgistics/time_parsers/american_datetime.rb

```diff
@@ -0,0 +1,30 @@
+module Newgistics
+  module TimeParsers
+    class AmericanDatetime
+      DATE_FORMAT = "%m/%d/%Y %I:%M %p %z".freeze
+
+      def parse(string)
+        date = date_with_timezone(string)
+        parse_date(date, string)
+      end
+
+      private
+
+      def date_with_timezone(string)
+        return string if includes_timezone?(string)
+        "#{string} #{Newgistics.time_zone.utc_offset}"
+      end
+
+      def includes_timezone?(string)
```

Comment:
just curious if we should create some sort of time helper or module for `date_with_timezone` and `includes_timezone?` could be useful in the future if there's more parsers. But also maybe just something to revisit if that happens

Reply:
Yeah that's a great point! I think we can leave it as-is for now and refactor if we introduce a third time parser that needs it as well